cmd/gen-go-sample/main_test.go

```diff
@@ -289,6 +289,35 @@ func TestWriteFile(t *testing.T) {
 
 }
 
+func TestEnum(t *testing.T) {
+	t.Parallel()
+
+	g := initTestGenerator()
+	vs := SampleValueSet{
+		ID: "my_value_set",
+		Parameters: SampleParameter{
+			Defaults: []string{
+				`e = FOO`,
```

Comment:
Could you use a unique name other than `FOO`? We already have a bunch of foos floating around, so this will make it easier to read and reason about. Thanks!

Reply:
Change `EType` to `FruitEnum`, and the value to `BANANA`.

Reply:
This helps a lot. Thank you!